### DISCUSSION TITLE
re-watching video annotations

### DISCUSSION THREAD

**Discussion by CeliaHanna** [biigle/core#570] [Q&A]
I notice that when I rewatch a video in BIIGLE that I have already annotated, the previous labels do not show up on the video but only on the time bar below it. Is it possible for the already made labels to be shown on the video itself so that I can be clear what I have already labelled in the video if I am watching it for a second time?

Many thanks

**Comment by mzur** [comment]
Video annotations that span multiple video frames ("annotation clips") are visible while the video is playing. Single-frame annotations are not. If you want to see the single-frame annotations again, you have to pause the video and select one of the annotations in the timeline. The video time then will jump to the exact position and the annotations will be shown.
